src/shared/components/query/QueryStore.ts

```diff
@@ -1458,11 +1450,12 @@ export class QueryStore
 				this._defaultStudySampleMap = studySampleMap;
 			}
 
-			const caseIds:string = (_window as any).serverVars.caseIds;
-			if (caseIds && (caseSetId === CUSTOM_CASE_LIST_ID)) {
-				this.caseIdsMode = 'sample';
-				this.caseIds = caseIds.split(/[\s+]/).join("\n");
-				this.initiallySelected.sampleListId = true;

```

Comment:
@adamabeshouse was the `this.initiallySelected.sampleListId` ok to remove?

Reply:
yes, its covered by https://github.com/cBioPortal/cbioportal-frontend/pull/1077/files/910aa3b80dc38b1c26884e73d4ede8770f3a58eb#diff-2edbc62f162a228681eff5291dc29be8R1444